src/platform/mhv/secondary-nav/components/MhvSecondaryNavMenu.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useFeatureToggle } from '~/platform/utilities/feature-toggles';
+import MhvSecondaryNavItem from './MhvSecondaryNavItem';
+
+/**
+ * MHV secondary navigation bar.
+ * LIMITATIONS: The logic to set an item to active is based on a first match approach
+ * looking through the list of items in reverse order. As such, '/my-health' home
+ * page is expected to be the first item in the list.
+ *
+ * @param {SecondaryNavItem[]} items the list of items to display in the navigation bar
+ * @returns the navigation bar
+ */
+const MhvSecondaryNavMenu = ({ items }) => {
+  const { useToggleValue, TOGGLE_NAMES } = useFeatureToggle();
+  const secNavEnabled = useToggleValue(TOGGLE_NAMES.mhvSecondaryNavEnabled);
+
+  if (secNavEnabled) {
+    // Perform a reverse find to match which nav link we are on, so we match on the home page last
+    const activeItem = [...items] // Clone the array, so the original stays the same
+      .reverse()
+      .find(item => window.location.pathname.startsWith(item.href));
+
+    const navContent = items.map(item => {
+      const key = item.title.toLowerCase().replaceAll(' ', '_');
+      return (
+        <MhvSecondaryNavItem
+          title={item.title}
+          href={item.href}
+          iconClass={item.iconClass}
+          abbreviation={item.abbreviation}
+          isActive={activeItem === item}
+          key={key}
+        />
+      );
+    });
+
+    return (
+      <nav id="mhv-sec-nav-bar" aria-label="My HealtheVet">
+        <div className="mhv-sec-nav-container vads-u-font-family--sans vads-font-weight-regular">
+          <div className="mhv-sec-nav-item-row">{navContent}</div>
+        </div>
+      </nav>
+    );
+  }
+  return null;
```

Comment:
Suggestion: Use a guard clause, here.

```
if (!enabled) {
  return null;
}
// feature toggle enabled code
```

Reply:
I did this initially with an else, but was flagged as unnecessary.

Reply:
If it had an `else`, it wouldn't be a "Guard clause".

Reply:
But guard clauses tend to be for nested conditionals. I don't understand how it helps here.

Reply:
If you search the codebase for `if (!` in `*.jsx` files, you will see many examples of using a guard clause. 

The benefit in this case is 1) returning early when we know this function doesn't need to output UI, 2) decreasing the indentation level of the nested code within the `if` statement, and 3) making the code within this component easier to read.

Reply:
I'll change it, but I do not see a real difference here.